spec/controllers/authors_controller_spec.rb

```diff
@@ -0,0 +1,44 @@
+require "rails_helper"
```

Comment:
I like the progression of the tests for this lab. I would like to to use the `assigns(:model)` pattern, though. (I know this is what Rails gives you if you have it stub out tests, so it's definitely a pattern you see often.) The test failures from these kinds of tests are not very descriptive of what's actually breaking. For example, this error is super confusing:

```
Failure/Error: expect(assigns(:author)).to be_new_record
  expected nil to be a new record, but was persisted
```

It's not the greatest pattern in the world (especially since it requires extra DB hits for all the tests), but I think it'll lend itself to clearer tests to do more of the setup in the actual tests, and check desired changes against a query of the DB. (More stuff similar to the test on line 16.)

Another example would be line 28...I'd change this to something more like:

```
let(:email) { 'test@example.com' }

before { post :create, email: email }

it "does not create" do
  expect(Author.find_by(email: email)).to be_nil
end
```